app/application/[menu]/[submenu]/page.js

```diff
@@ -11,6 +12,8 @@ export default async function SubAppPages({ params }) {
       return <Project data={tasks} projectId={params.submenu} />;
     case "setting":
       return <Today data={tasks} settingMenu={params.submenu} />;
+    case "task":
+      return <Today data={tasks} taskId={params.submenu} />;
     default:
       notFound();
```

Comment:
![image](https://github.com/zli213/SynergyStudy/assets/119087910/004ed449-b2b9-471f-b8c2-628f894a9af0)
I see the Todoist use "Task not found" when input wrong task id in URL, so I tried to make it work. Now if I change the submenu to something else, it will show "Task not found". If I change the task id, it will show the 404. 
Still need to fix it.